[2022-02-16 18:01:53] request.INFO: Matched route "api_translation_value_edit". {"route":"api_translation_value_edit","route_parameters":{"_controller":"prestashop.core.api.translation.controller:translationEditAction","_legacy_controller":"AdminTranslations","_route":"api_translation_value_edit"},"request_uri":"https://brasas.ec/gqth8jpeqegdu5rw/index.php/api/translations/edit?_token=Xtx6p41NUpjShGYnELN_lps9DHklJ00ygcEbMw6ji-k","method":"POST"} []
[2022-02-16 18:01:53] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2022-02-16 18:01:53] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"santiago@brasas.ec"} []
[2022-02-16 18:01:54] app.ERROR: No result was found for query although at least one row was expected. {"object_type":"Translation"} []
[2022-02-16 18:01:54] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u506993268/domains/brasas.ec/public_html/var/cache/prod/pools/bNLE+cN-bh/): failed to open dir: No such file or directory at /home/u506993268/domains/brasas.ec/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2022-02-16 18:01:54] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u506993268/domains/brasas.ec/public_html/var/cache/prod/pools/MQOVCZDjF6/): failed to open dir: No such file or directory at /home/u506993268/domains/brasas.ec/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2022-02-16 18:01:54] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u506993268/domains/brasas.ec/public_html/var/cache/prod/pools/6e3eG+QUXB/): failed to open dir: No such file or directory at /home/u506993268/domains/brasas.ec/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2022-02-16 18:01:54] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u506993268/domains/brasas.ec/public_html/var/cache/prod/pools/qnOg9807ad/): failed to open dir: No such file or directory at /home/u506993268/domains/brasas.ec/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2022-02-16 18:01:54] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2022-02-16 18:02:15] request.INFO: Matched route "api_translation_value_edit". {"route":"api_translation_value_edit","route_parameters":{"_controller":"prestashop.core.api.translation.controller:translationEditAction","_legacy_controller":"AdminTranslations","_route":"api_translation_value_edit"},"request_uri":"https://brasas.ec/gqth8jpeqegdu5rw/index.php/api/translations/edit?_token=Xtx6p41NUpjShGYnELN_lps9DHklJ00ygcEbMw6ji-k","method":"POST"} []
[2022-02-16 18:02:15] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2022-02-16 18:02:15] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"santiago@brasas.ec"} []
[2022-02-16 18:02:15] app.ERROR: No result was found for query although at least one row was expected. {"object_type":"Translation"} []
[2022-02-16 18:02:15] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u506993268/domains/brasas.ec/public_html/var/cache/prod/pools/bNLE+cN-bh/): failed to open dir: No such file or directory at /home/u506993268/domains/brasas.ec/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2022-02-16 18:02:15] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u506993268/domains/brasas.ec/public_html/var/cache/prod/pools/MQOVCZDjF6/): failed to open dir: No such file or directory at /home/u506993268/domains/brasas.ec/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2022-02-16 18:02:15] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u506993268/domains/brasas.ec/public_html/var/cache/prod/pools/6e3eG+QUXB/): failed to open dir: No such file or directory at /home/u506993268/domains/brasas.ec/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2022-02-16 18:02:15] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u506993268/domains/brasas.ec/public_html/var/cache/prod/pools/qnOg9807ad/): failed to open dir: No such file or directory at /home/u506993268/domains/brasas.ec/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2022-02-16 18:02:15] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2022-02-17 08:18:27] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException: "No route found for "GET /sell/orders/place": Method Not Allowed (Allow: POST)" at /home/u506993268/domains/brasas.ec/public_html/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 141 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\MethodNotAllowedHttpException(code: 0): No route found for \"GET /sell/orders/place\": Method Not Allowed (Allow: POST) at /home/u506993268/domains/brasas.ec/public_html/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/EventListener/RouterListener.php:141, Symfony\\Component\\Routing\\Exception\\MethodNotAllowedException(code: 0):  at /home/u506993268/domains/brasas.ec/public_html/var/cache/prod/appProdProjectContainerUrlMatcher.php:7602)"} []
[2022-02-21 20:22:04] request.INFO: Matched route "admin_module_import". {"route":"admin_module_import","route_parameters":{"module_name":null,"_controller":"PrestaShopBundle\\Controller\\Admin\\Improve\\ModuleController::importModuleAction","_route":"admin_module_import"},"request_uri":"https://brasas.ec/gqth8jpeqegdu5rw/index.php/improve/modules/import?_token=Am3dTG6yw6F1rv1j5MxgmQEcpO2BaYbukjTqSIDYM4Q","method":"POST"} []
[2022-02-21 20:22:04] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2022-02-21 20:22:04] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"santiago@brasas.ec"} []
[2022-02-21 20:22:13] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\ClassNotFoundException: "Attempted to load class "HTTPCache" from namespace "PrestaShop\Module\PerformancePro\domain\service\cache". Did you forget a "use" statement for "PrestaShop\Module\PerformancePro\domain\service\cache\HTTPCache"?" at /home/u506993268/domains/brasas.ec/public_html/modules/performancepro/performancepro.php line 62 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\ClassNotFoundException(code: 0): Attempted to load class \"HTTPCache\" from namespace \"PrestaShop\\Module\\PerformancePro\\domain\\service\\cache\".\nDid you forget a \"use\" statement for \"PrestaShop\\Module\\PerformancePro\\domain\\service\\cache\\HTTPCache\"? at /home/u506993268/domains/brasas.ec/public_html/modules/performancepro/performancepro.php:62)"} []
[2022-02-21 20:22:13] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2022-02-24 21:55:51] request.INFO: Matched route "api_translation_value_edit". {"route":"api_translation_value_edit","route_parameters":{"_controller":"prestashop.core.api.translation.controller:translationEditAction","_legacy_controller":"AdminTranslations","_route":"api_translation_value_edit"},"request_uri":"https://brasas.ec/gqth8jpeqegdu5rw/index.php/api/translations/edit?_token=Nt_dgd76MtPJeO0p9QTPvOGwbn1j0mYV4bRNZUJ3tNw","method":"POST"} []
[2022-02-24 21:55:51] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2022-02-24 21:55:51] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"santiago@brasas.ec"} []
[2022-02-24 21:55:51] app.ERROR: No result was found for query although at least one row was expected. {"object_type":"Translation"} []
[2022-02-24 21:55:52] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u506993268/domains/brasas.ec/public_html/var/cache/prod/pools/bNLE+cN-bh/): failed to open dir: No such file or directory at /home/u506993268/domains/brasas.ec/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2022-02-24 21:55:52] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u506993268/domains/brasas.ec/public_html/var/cache/prod/pools/MQOVCZDjF6/): failed to open dir: No such file or directory at /home/u506993268/domains/brasas.ec/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2022-02-24 21:55:52] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u506993268/domains/brasas.ec/public_html/var/cache/prod/pools/6e3eG+QUXB/): failed to open dir: No such file or directory at /home/u506993268/domains/brasas.ec/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2022-02-24 21:55:52] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u506993268/domains/brasas.ec/public_html/var/cache/prod/pools/qnOg9807ad/): failed to open dir: No such file or directory at /home/u506993268/domains/brasas.ec/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2022-02-24 21:55:52] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2022-02-24 22:00:01] request.INFO: Matched route "api_translation_value_edit". {"route":"api_translation_value_edit","route_parameters":{"_controller":"prestashop.core.api.translation.controller:translationEditAction","_legacy_controller":"AdminTranslations","_route":"api_translation_value_edit"},"request_uri":"https://brasas.ec/gqth8jpeqegdu5rw/index.php/api/translations/edit?_token=Nt_dgd76MtPJeO0p9QTPvOGwbn1j0mYV4bRNZUJ3tNw","method":"POST"} []
[2022-02-24 22:00:01] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2022-02-24 22:00:01] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"santiago@brasas.ec"} []
[2022-02-24 22:00:01] app.ERROR: No result was found for query although at least one row was expected. {"object_type":"Translation"} []
[2022-02-24 22:00:01] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u506993268/domains/brasas.ec/public_html/var/cache/prod/pools/bNLE+cN-bh/): failed to open dir: No such file or directory at /home/u506993268/domains/brasas.ec/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2022-02-24 22:00:01] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u506993268/domains/brasas.ec/public_html/var/cache/prod/pools/MQOVCZDjF6/): failed to open dir: No such file or directory at /home/u506993268/domains/brasas.ec/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2022-02-24 22:00:01] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u506993268/domains/brasas.ec/public_html/var/cache/prod/pools/6e3eG+QUXB/): failed to open dir: No such file or directory at /home/u506993268/domains/brasas.ec/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2022-02-24 22:00:01] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u506993268/domains/brasas.ec/public_html/var/cache/prod/pools/qnOg9807ad/): failed to open dir: No such file or directory at /home/u506993268/domains/brasas.ec/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2022-02-24 22:00:01] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2022-05-26 12:04:42] request.INFO: Matched route "admin_module_manage_action". {"route":"admin_module_manage_action","route_parameters":{"_controller":"PrestaShopBundle\\Controller\\Admin\\Improve\\ModuleController::moduleAction","action":"upgrade","module_name":"statssales","_route":"admin_module_manage_action"},"request_uri":"https://dev.velox.ec/zonatech/gqth8jpeqegdu5rw/index.php/improve/modules/manage/action/upgrade/statssales?_token=Xq9o542Bgb5Vfv87MVbfY38wuOgh5NOaAOjJuXP-cPY","method":"POST"} []
[2022-05-26 12:04:44] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2022-05-26 12:04:44] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"soporte@velox.ec"} []
[2022-05-26 12:04:49] app.ERROR: Excepción emitida por el módulo statssales en upgrade. Error sent by Addons. You may need to be logged. {"object_type":"Module","object_id":53} []
[2022-05-26 12:04:49] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2022-06-12 21:46:34] request.INFO: Matched route "api_translation_value_edit". {"route":"api_translation_value_edit","route_parameters":{"_controller":"prestashop.core.api.translation.controller:translationEditAction","_legacy_controller":"AdminTranslations","_route":"api_translation_value_edit"},"request_uri":"https://legadoec.com/gqth8jpeqegdu5rw/index.php/api/translations/edit?_token=lXkp2qp6mf1NAYqoJdVtFR48YCa83c5fjGqHSm8ZBq4","method":"POST"} []
[2022-06-12 21:46:34] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2022-06-12 21:46:34] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"soporte@velox.ec"} []
[2022-06-12 21:46:34] app.ERROR: No result was found for query although at least one row was expected. {"object_type":"Translation"} []
[2022-06-12 21:46:34] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u379649439/domains/legadoec.com/public_html/var/cache/prod/pools/ZwcenjAksE/): failed to open dir: No such file or directory at /home/u379649439/domains/legadoec.com/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2022-06-12 21:46:34] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u379649439/domains/legadoec.com/public_html/var/cache/prod/pools/HyGCLfBC9f/): failed to open dir: No such file or directory at /home/u379649439/domains/legadoec.com/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2022-06-12 21:46:34] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u379649439/domains/legadoec.com/public_html/var/cache/prod/pools/i-dZ-CgpXa/): failed to open dir: No such file or directory at /home/u379649439/domains/legadoec.com/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2022-06-12 21:46:34] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u379649439/domains/legadoec.com/public_html/var/cache/prod/pools/r2dco7nlq9/): failed to open dir: No such file or directory at /home/u379649439/domains/legadoec.com/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2022-06-12 21:46:34] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2022-06-29 13:26:35] request.INFO: Matched route "admin_product_form". {"route":"admin_product_form","route_parameters":{"_controller":"PrestaShopBundle\\Controller\\Admin\\ProductController::formAction","_legacy_controller":"AdminProducts","_legacy_param_mapper_class":"PrestaShop\\PrestaShop\\Adapter\\Product\\AdminProductDataProvider","_legacy_param_mapper_method":"mapLegacyParametersProductForm","id":"53","_route":"admin_product_form"},"request_uri":"https://legadoec.com/gqth8jpeqegdu5rw/index.php/sell/catalog/products/53?_token=B_Z3_sx9I9UDXv4GuorX4ONih_KQuzSKyH7BMPo2YU4","method":"GET"} []
[2022-06-29 13:26:35] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2022-06-29 13:26:35] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"info@legadoec.com"} []
[2022-06-29 13:26:42] app.ERROR: Data from PrestaShop Addons is invalid, and cannot fallback on cache. [] []
[2022-06-29 13:26:43] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2023-01-04 12:17:01] request.INFO: Matched route "api_translation_value_edit". {"route":"api_translation_value_edit","route_parameters":{"_controller":"prestashop.core.api.translation.controller:translationEditAction","_legacy_controller":"AdminTranslations","_route":"api_translation_value_edit"},"request_uri":"https://dev.velox.ec/mom/gqth8jpeqegdu5rw/index.php/api/translations/edit?_token=UfdHptZx-xoKcXEZVs8ogAK8fG9vIOBx_ZGIZbSFDn4","method":"POST"} []
[2023-01-04 12:17:01] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2023-01-04 12:17:01] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"soporte@velox.ec"} []
[2023-01-04 12:17:01] app.ERROR: No result was found for query although at least one row was expected. {"object_type":"Translation"} []
[2023-01-04 12:17:02] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u805614560/domains/dev.velox.ec/public_html/mom/var/cache/prod/pools/u438wU5s9A/): failed to open dir: No such file or directory at /home/u805614560/domains/dev.velox.ec/public_html/mom/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-01-04 12:17:02] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u805614560/domains/dev.velox.ec/public_html/mom/var/cache/prod/pools/Y3nJa2wlql/): failed to open dir: No such file or directory at /home/u805614560/domains/dev.velox.ec/public_html/mom/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-01-04 12:17:02] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u805614560/domains/dev.velox.ec/public_html/mom/var/cache/prod/pools/qJRfebLOTM/): failed to open dir: No such file or directory at /home/u805614560/domains/dev.velox.ec/public_html/mom/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-01-04 12:17:02] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u805614560/domains/dev.velox.ec/public_html/mom/var/cache/prod/pools/vJKJ6s3KQH/): failed to open dir: No such file or directory at /home/u805614560/domains/dev.velox.ec/public_html/mom/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-01-04 12:17:02] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2023-01-04 12:17:32] request.INFO: Matched route "api_translation_value_edit". {"route":"api_translation_value_edit","route_parameters":{"_controller":"prestashop.core.api.translation.controller:translationEditAction","_legacy_controller":"AdminTranslations","_route":"api_translation_value_edit"},"request_uri":"https://dev.velox.ec/mom/gqth8jpeqegdu5rw/index.php/api/translations/edit?_token=UfdHptZx-xoKcXEZVs8ogAK8fG9vIOBx_ZGIZbSFDn4","method":"POST"} []
[2023-01-04 12:17:32] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2023-01-04 12:17:32] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"soporte@velox.ec"} []
[2023-01-04 12:17:32] app.ERROR: No result was found for query although at least one row was expected. {"object_type":"Translation"} []
[2023-01-04 12:17:33] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u805614560/domains/dev.velox.ec/public_html/mom/var/cache/prod/pools/u438wU5s9A/): failed to open dir: No such file or directory at /home/u805614560/domains/dev.velox.ec/public_html/mom/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-01-04 12:17:33] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u805614560/domains/dev.velox.ec/public_html/mom/var/cache/prod/pools/Y3nJa2wlql/): failed to open dir: No such file or directory at /home/u805614560/domains/dev.velox.ec/public_html/mom/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-01-04 12:17:33] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u805614560/domains/dev.velox.ec/public_html/mom/var/cache/prod/pools/qJRfebLOTM/): failed to open dir: No such file or directory at /home/u805614560/domains/dev.velox.ec/public_html/mom/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-01-04 12:17:33] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u805614560/domains/dev.velox.ec/public_html/mom/var/cache/prod/pools/vJKJ6s3KQH/): failed to open dir: No such file or directory at /home/u805614560/domains/dev.velox.ec/public_html/mom/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-01-04 12:17:33] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2023-03-04 14:29:04] request.INFO: Matched route "api_translation_value_edit". {"route":"api_translation_value_edit","route_parameters":{"_controller":"prestashop.core.api.translation.controller:translationEditAction","_legacy_controller":"AdminTranslations","_route":"api_translation_value_edit"},"request_uri":"https://dev.velox.ec/mom/gqth8jpeqegdu5rw/index.php/api/translations/edit?_token=11_l3jS0wQPpjtfvqQidS8MhTwbNIwRG8PAU6bX1vg0","method":"POST"} []
[2023-03-04 14:29:04] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2023-03-04 14:29:04] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"soporte@velox.ec"} []
[2023-03-04 14:29:04] app.ERROR: No result was found for query although at least one row was expected. {"object_type":"Translation"} []
[2023-03-04 14:29:04] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u805614560/domains/dev.velox.ec/public_html/mom/var/cache/prod/pools/u438wU5s9A/): failed to open dir: No such file or directory at /home/u805614560/domains/dev.velox.ec/public_html/mom/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-03-04 14:29:05] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u805614560/domains/dev.velox.ec/public_html/mom/var/cache/prod/pools/Y3nJa2wlql/): failed to open dir: No such file or directory at /home/u805614560/domains/dev.velox.ec/public_html/mom/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-03-04 14:29:05] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u805614560/domains/dev.velox.ec/public_html/mom/var/cache/prod/pools/qJRfebLOTM/): failed to open dir: No such file or directory at /home/u805614560/domains/dev.velox.ec/public_html/mom/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-03-04 14:29:05] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u805614560/domains/dev.velox.ec/public_html/mom/var/cache/prod/pools/vJKJ6s3KQH/): failed to open dir: No such file or directory at /home/u805614560/domains/dev.velox.ec/public_html/mom/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-03-04 14:29:05] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2023-03-04 14:31:10] request.INFO: Matched route "api_translation_value_edit". {"route":"api_translation_value_edit","route_parameters":{"_controller":"prestashop.core.api.translation.controller:translationEditAction","_legacy_controller":"AdminTranslations","_route":"api_translation_value_edit"},"request_uri":"https://dev.velox.ec/mom/gqth8jpeqegdu5rw/index.php/api/translations/edit?_token=11_l3jS0wQPpjtfvqQidS8MhTwbNIwRG8PAU6bX1vg0","method":"POST"} []
[2023-03-04 14:31:10] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2023-03-04 14:31:10] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"soporte@velox.ec"} []
[2023-03-04 14:31:10] app.ERROR: No result was found for query although at least one row was expected. {"object_type":"Translation"} []
[2023-03-04 14:31:10] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u805614560/domains/dev.velox.ec/public_html/mom/var/cache/prod/pools/u438wU5s9A/): failed to open dir: No such file or directory at /home/u805614560/domains/dev.velox.ec/public_html/mom/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-03-04 14:31:10] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u805614560/domains/dev.velox.ec/public_html/mom/var/cache/prod/pools/Y3nJa2wlql/): failed to open dir: No such file or directory at /home/u805614560/domains/dev.velox.ec/public_html/mom/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-03-04 14:31:10] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u805614560/domains/dev.velox.ec/public_html/mom/var/cache/prod/pools/qJRfebLOTM/): failed to open dir: No such file or directory at /home/u805614560/domains/dev.velox.ec/public_html/mom/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-03-04 14:31:10] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u805614560/domains/dev.velox.ec/public_html/mom/var/cache/prod/pools/vJKJ6s3KQH/): failed to open dir: No such file or directory at /home/u805614560/domains/dev.velox.ec/public_html/mom/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-03-04 14:31:10] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2023-03-04 16:24:26] request.INFO: Matched route "admin_module_import". {"route":"admin_module_import","route_parameters":{"module_name":null,"_controller":"PrestaShopBundle\\Controller\\Admin\\Improve\\ModuleController::importModuleAction","_route":"admin_module_import"},"request_uri":"https://dev.velox.ec/mom/gqth8jpeqegdu5rw/index.php/improve/modules/import?_token=CVKcrz3jir1fvO9NFu2BF52Sews8lZRaEm9IQrw5Q_I","method":"POST"} []
[2023-03-04 16:24:26] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2023-03-04 16:24:26] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"soporte@velox.ec"} []
[2023-03-04 16:24:26] app.INFO: Protect vendor folder in module creativepopup [] []
[2023-03-04 16:24:26] app.INFO: Module creativepopup has no vendor folder [] []
[2023-03-04 16:24:26] app.ERROR: Missing class name of tab [] []
[2023-03-04 16:24:26] app.ERROR: Missing class name of tab [] []
[2023-03-04 16:24:26] app.ERROR: Missing class name of tab [] []
[2023-03-04 16:24:26] app.ERROR: Missing class name of tab [] []
[2023-03-04 16:24:26] app.ERROR: Missing class name of tab [] []
[2023-03-04 16:24:26] app.ERROR: Missing class name of tab [] []
[2023-03-04 16:24:26] app.ERROR: Missing class name of tab [] []
[2023-03-04 16:24:26] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2023-03-04 16:31:08] request.CRITICAL: Uncaught PHP Exception PrestaShop\PrestaShop\Core\Localization\Exception\LocalizationException: "Invalid $number parameter: "" cannot be interpreted as a number" at /home/u805614560/domains/dev.velox.ec/public_html/mom/src/Core/Localization/Number/Formatter.php line 108 {"exception":"[object] (PrestaShop\\PrestaShop\\Core\\Localization\\Exception\\LocalizationException(code: 0): Invalid $number parameter: \"\" cannot be interpreted as a number at /home/u805614560/domains/dev.velox.ec/public_html/mom/src/Core/Localization/Number/Formatter.php:108, InvalidArgumentException(code: 0): \"\" cannot be interpreted as a number at /home/u805614560/domains/dev.velox.ec/public_html/mom/vendor/prestashop/decimal/src/Builder.php:40)"} []
[2023-03-04 16:41:31] request.CRITICAL: Uncaught PHP Exception PrestaShop\PrestaShop\Core\Localization\Exception\LocalizationException: "Invalid $number parameter: "" cannot be interpreted as a number" at /home/u805614560/domains/dev.velox.ec/public_html/mom/src/Core/Localization/Number/Formatter.php line 108 {"exception":"[object] (PrestaShop\\PrestaShop\\Core\\Localization\\Exception\\LocalizationException(code: 0): Invalid $number parameter: \"\" cannot be interpreted as a number at /home/u805614560/domains/dev.velox.ec/public_html/mom/src/Core/Localization/Number/Formatter.php:108, InvalidArgumentException(code: 0): \"\" cannot be interpreted as a number at /home/u805614560/domains/dev.velox.ec/public_html/mom/vendor/prestashop/decimal/src/Builder.php:40)"} []
[2023-08-06 16:31:07] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalThrowableError: "Call to a member function getLocale() on null" at /home/u672279739/domains/salvacerohomecenter.com/public_html/classes/controller/Controller.php line 196 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Call to a member function getLocale() on null at /home/u672279739/domains/salvacerohomecenter.com/public_html/classes/controller/Controller.php:196)"} []
[2023-08-06 16:32:11] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalThrowableError: "Call to a member function getLocale() on null" at /home/u672279739/domains/salvacerohomecenter.com/public_html/classes/controller/Controller.php line 196 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Call to a member function getLocale() on null at /home/u672279739/domains/salvacerohomecenter.com/public_html/classes/controller/Controller.php:196)"} []
[2023-08-06 16:33:23] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalThrowableError: "Call to a member function getLocale() on null" at /home/u672279739/domains/salvacerohomecenter.com/public_html/classes/controller/Controller.php line 196 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Call to a member function getLocale() on null at /home/u672279739/domains/salvacerohomecenter.com/public_html/classes/controller/Controller.php:196)"} []
[2023-08-06 16:36:34] request.INFO: Matched route "admin_customers_view". {"route":"admin_customers_view","route_parameters":{"_controller":"PrestaShopBundle\\Controller\\Admin\\Sell\\Customer\\CustomerController::viewAction","_legacy_controller":"AdminCustomers","_legacy_link":"AdminCustomers:viewcustomer","_legacy_parameters":{"id_customer":"customerId"},"customerId":"2","_route":"admin_customers_view"},"request_uri":"https://salvacerohomecenter.com/gqth8jpeqegdu5rw/index.php/sell/customers/2/view?_token=6a4bPZEwwAM0RXIGsV6LOeQHOTMkcpjiLD3Y8CARGQs","method":"GET"} []
[2023-08-06 16:36:34] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2023-08-06 16:36:34] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"invitado@velox.ec"} []
[2023-08-06 16:36:34] request.CRITICAL: Uncaught PHP Exception Twig\Error\RuntimeError: "An exception has been thrown during the rendering of a template ("Unable to load template 'module:salvacero_functions/views/templates/hook/amount.tpl'")." at /home/u672279739/domains/salvacerohomecenter.com/public_html/src/PrestaShopBundle/Resources/views/Admin/Sell/Customer/view.html.twig line 52 {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): An exception has been thrown during the rendering of a template (\"Unable to load template 'module:salvacero_functions/views/templates/hook/amount.tpl'\"). at /home/u672279739/domains/salvacerohomecenter.com/public_html/src/PrestaShopBundle/Resources/views/Admin/Sell/Customer/view.html.twig:52, SmartyException(code: 0): Unable to load template 'module:salvacero_functions/views/templates/hook/amount.tpl' at /home/u672279739/domains/salvacerohomecenter.com/public_html/vendor/smarty/smarty/libs/sysplugins/smarty_internal_template.php:195)"} []
[2023-08-06 16:36:34] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2023-08-07 01:57:12] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Routing\Exception\RouteNotFoundException: "Unable to generate a URL for the named route "1" as such route does not exist." at /home/u672279739/domains/salvacerohomecenter.com/public_html/var/cache/prod/appProdProjectContainerUrlGenerator.php line 591 {"exception":"[object] (Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"1\" as such route does not exist. at /home/u672279739/domains/salvacerohomecenter.com/public_html/var/cache/prod/appProdProjectContainerUrlGenerator.php:591)"} []
[2023-08-07 02:05:34] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Routing\Exception\MissingMandatoryParametersException: "Some mandatory parameters are missing ("customerId") to generate a URL for route "admin_customers_view"." at /home/u672279739/domains/salvacerohomecenter.com/public_html/vendor/symfony/symfony/src/Symfony/Component/Routing/Generator/UrlGenerator.php line 136 {"exception":"[object] (Symfony\\Component\\Routing\\Exception\\MissingMandatoryParametersException(code: 0): Some mandatory parameters are missing (\"customerId\") to generate a URL for route \"admin_customers_view\". at /home/u672279739/domains/salvacerohomecenter.com/public_html/vendor/symfony/symfony/src/Symfony/Component/Routing/Generator/UrlGenerator.php:136)"} []
[2023-08-28 21:58:08] request.INFO: Matched route "api_translation_value_edit". {"route":"api_translation_value_edit","route_parameters":{"_controller":"prestashop.core.api.translation.controller:translationEditAction","_legacy_controller":"AdminTranslations","_route":"api_translation_value_edit"},"request_uri":"https://salvacerohomecenter.com/gqth8jpeqegdu5rw/index.php/api/translations/edit?_token=LoN6oWAT37vp5ptg0Tt8LSdvY_xbWa8zGVZ5Tpn65Fo","method":"POST"} []
[2023-08-28 21:58:08] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2023-08-28 21:58:08] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"soporte@velox.ec"} []
[2023-08-28 21:58:08] app.ERROR: No result was found for query although at least one row was expected. {"object_type":"Translation"} []
[2023-08-28 21:58:08] app.ERROR: No result was found for query although at least one row was expected. {"object_type":"Translation"} []
[2023-08-28 21:58:08] app.ERROR: No result was found for query although at least one row was expected. {"object_type":"Translation"} []
[2023-08-28 21:58:09] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u672279739/domains/salvacerohomecenter.com/public_html/var/cache/prod/pools/wKHidGbpND/): failed to open dir: No such file or directory at /home/u672279739/domains/salvacerohomecenter.com/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-08-28 21:58:09] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u672279739/domains/salvacerohomecenter.com/public_html/var/cache/prod/pools/4-rXROHpC6/): failed to open dir: No such file or directory at /home/u672279739/domains/salvacerohomecenter.com/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-08-28 21:58:09] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u672279739/domains/salvacerohomecenter.com/public_html/var/cache/prod/pools/rvNBXm14bF/): failed to open dir: No such file or directory at /home/u672279739/domains/salvacerohomecenter.com/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-08-28 21:58:09] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u672279739/domains/salvacerohomecenter.com/public_html/var/cache/prod/pools/Bxo9eUMTV3/): failed to open dir: No such file or directory at /home/u672279739/domains/salvacerohomecenter.com/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-08-28 21:58:09] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2023-12-06 09:09:47] request.INFO: Matched route "api_translation_value_edit". {"route":"api_translation_value_edit","route_parameters":{"_controller":"prestashop.core.api.translation.controller:translationEditAction","_legacy_controller":"AdminTranslations","_route":"api_translation_value_edit"},"request_uri":"https://salvacerohomecenter.com/gqth8jpeqegdu5rw/index.php/api/translations/edit?_token=eJnSNvds7ElX3v3sQEVE8SVp3XLcmmZQzS6J96hezaM","method":"POST"} []
[2023-12-06 09:09:47] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2023-12-06 09:09:47] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"soporte@velox.ec"} []
[2023-12-06 09:09:47] app.ERROR: No result was found for query although at least one row was expected. {"object_type":"Translation"} []
[2023-12-06 09:09:48] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u672279739/domains/salvacerohomecenter.com/public_html/var/cache/prod/pools/wKHidGbpND/): failed to open dir: No such file or directory at /home/u672279739/domains/salvacerohomecenter.com/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-12-06 09:09:48] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u672279739/domains/salvacerohomecenter.com/public_html/var/cache/prod/pools/4-rXROHpC6/): failed to open dir: No such file or directory at /home/u672279739/domains/salvacerohomecenter.com/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-12-06 09:09:48] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u672279739/domains/salvacerohomecenter.com/public_html/var/cache/prod/pools/rvNBXm14bF/): failed to open dir: No such file or directory at /home/u672279739/domains/salvacerohomecenter.com/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-12-06 09:09:48] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u672279739/domains/salvacerohomecenter.com/public_html/var/cache/prod/pools/Bxo9eUMTV3/): failed to open dir: No such file or directory at /home/u672279739/domains/salvacerohomecenter.com/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-12-06 09:09:48] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2023-12-06 09:11:09] request.INFO: Matched route "api_translation_value_edit". {"route":"api_translation_value_edit","route_parameters":{"_controller":"prestashop.core.api.translation.controller:translationEditAction","_legacy_controller":"AdminTranslations","_route":"api_translation_value_edit"},"request_uri":"https://salvacerohomecenter.com/gqth8jpeqegdu5rw/index.php/api/translations/edit?_token=eJnSNvds7ElX3v3sQEVE8SVp3XLcmmZQzS6J96hezaM","method":"POST"} []
[2023-12-06 09:11:09] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2023-12-06 09:11:09] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"soporte@velox.ec"} []
[2023-12-06 09:11:09] app.ERROR: No result was found for query although at least one row was expected. {"object_type":"Translation"} []
[2023-12-06 09:11:09] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u672279739/domains/salvacerohomecenter.com/public_html/var/cache/prod/pools/wKHidGbpND/): failed to open dir: No such file or directory at /home/u672279739/domains/salvacerohomecenter.com/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-12-06 09:11:09] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u672279739/domains/salvacerohomecenter.com/public_html/var/cache/prod/pools/4-rXROHpC6/): failed to open dir: No such file or directory at /home/u672279739/domains/salvacerohomecenter.com/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-12-06 09:11:09] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u672279739/domains/salvacerohomecenter.com/public_html/var/cache/prod/pools/rvNBXm14bF/): failed to open dir: No such file or directory at /home/u672279739/domains/salvacerohomecenter.com/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-12-06 09:11:09] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u672279739/domains/salvacerohomecenter.com/public_html/var/cache/prod/pools/Bxo9eUMTV3/): failed to open dir: No such file or directory at /home/u672279739/domains/salvacerohomecenter.com/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-12-06 09:11:09] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2023-12-06 09:11:22] request.INFO: Matched route "api_translation_value_edit". {"route":"api_translation_value_edit","route_parameters":{"_controller":"prestashop.core.api.translation.controller:translationEditAction","_legacy_controller":"AdminTranslations","_route":"api_translation_value_edit"},"request_uri":"https://salvacerohomecenter.com/gqth8jpeqegdu5rw/index.php/api/translations/edit?_token=eJnSNvds7ElX3v3sQEVE8SVp3XLcmmZQzS6J96hezaM","method":"POST"} []
[2023-12-06 09:11:22] security.DEBUG: Read existing security token from the session. {"key":"_security_main","token_class":"Symfony\\Component\\Security\\Core\\Authentication\\Token\\UsernamePasswordToken"} []
[2023-12-06 09:11:22] security.DEBUG: User was reloaded from a user provider. {"provider":"PrestaShopBundle\\Security\\Admin\\EmployeeProvider","username":"soporte@velox.ec"} []
[2023-12-06 09:11:22] app.ERROR: No result was found for query although at least one row was expected. {"object_type":"Translation"} []
[2023-12-06 09:11:22] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u672279739/domains/salvacerohomecenter.com/public_html/var/cache/prod/pools/wKHidGbpND/): failed to open dir: No such file or directory at /home/u672279739/domains/salvacerohomecenter.com/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-12-06 09:11:22] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u672279739/domains/salvacerohomecenter.com/public_html/var/cache/prod/pools/4-rXROHpC6/): failed to open dir: No such file or directory at /home/u672279739/domains/salvacerohomecenter.com/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-12-06 09:11:22] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u672279739/domains/salvacerohomecenter.com/public_html/var/cache/prod/pools/rvNBXm14bF/): failed to open dir: No such file or directory at /home/u672279739/domains/salvacerohomecenter.com/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-12-06 09:11:22] cache.WARNING: Failed to clear the cache {"exception":"[object] (UnexpectedValueException(code: 0): RecursiveDirectoryIterator::__construct(/home/u672279739/domains/salvacerohomecenter.com/public_html/var/cache/prod/pools/Bxo9eUMTV3/): failed to open dir: No such file or directory at /home/u672279739/domains/salvacerohomecenter.com/public_html/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/FilesystemCommonTrait.php:58)"} []
[2023-12-06 09:11:22] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2023-12-27 23:08:09] request.CRITICAL: Uncaught PHP Exception Exception: "Cannot find role slug" at /home/u672279739/domains/salvacerohomecenter.com/public_html/classes/Access.php line 376 {"exception":"[object] (Exception(code: 0): Cannot find role slug at /home/u672279739/domains/salvacerohomecenter.com/public_html/classes/Access.php:376)"} []
[2024-03-22 14:48:30] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\DBALException: "An exception occured while establishing a connection to figure out your platform version. You can circumvent this by setting a 'server_version' configuration value  For further information have a look at: https://github.com/doctrine/DoctrineBundle/issues/673" at C:\xampp\htdocs\svecommerce\vendor\doctrine\doctrine-bundle\ConnectionFactory.php line 79 {"exception":"[object] (Doctrine\\DBAL\\DBALException(code: 0): An exception occured while establishing a connection to figure out your platform version.\r\nYou can circumvent this by setting a 'server_version' configuration value\r\n\r\nFor further information have a look at:\r\nhttps://github.com/doctrine/DoctrineBundle/issues/673 at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\doctrine-bundle\\ConnectionFactory.php:79, Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:93, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:31, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:27)"} []
[2024-03-22 14:48:30] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\DBALException: An exception occured while establishing a connection to figure out your platform version. You can circumvent this by setting a 'server_version' configuration value  For further information have a look at: https://github.com/doctrine/DoctrineBundle/issues/673 at C:\xampp\htdocs\svecommerce\vendor\doctrine\doctrine-bundle\ConnectionFactory.php line 79) {"exception":"[object] (Doctrine\\DBAL\\DBALException(code: 0): An exception occured while establishing a connection to figure out your platform version.\r\nYou can circumvent this by setting a 'server_version' configuration value\r\n\r\nFor further information have a look at:\r\nhttps://github.com/doctrine/DoctrineBundle/issues/673 at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\doctrine-bundle\\ConnectionFactory.php:79, Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:93, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:31, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:27)"} []
[2024-03-22 14:49:10] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\DBALException: "An exception occured while establishing a connection to figure out your platform version. You can circumvent this by setting a 'server_version' configuration value  For further information have a look at: https://github.com/doctrine/DoctrineBundle/issues/673" at C:\xampp\htdocs\svecommerce\vendor\doctrine\doctrine-bundle\ConnectionFactory.php line 79 {"exception":"[object] (Doctrine\\DBAL\\DBALException(code: 0): An exception occured while establishing a connection to figure out your platform version.\r\nYou can circumvent this by setting a 'server_version' configuration value\r\n\r\nFor further information have a look at:\r\nhttps://github.com/doctrine/DoctrineBundle/issues/673 at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\doctrine-bundle\\ConnectionFactory.php:79, Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:93, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:31, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:27)"} []
[2024-03-22 14:49:10] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\DBALException: An exception occured while establishing a connection to figure out your platform version. You can circumvent this by setting a 'server_version' configuration value  For further information have a look at: https://github.com/doctrine/DoctrineBundle/issues/673 at C:\xampp\htdocs\svecommerce\vendor\doctrine\doctrine-bundle\ConnectionFactory.php line 79) {"exception":"[object] (Doctrine\\DBAL\\DBALException(code: 0): An exception occured while establishing a connection to figure out your platform version.\r\nYou can circumvent this by setting a 'server_version' configuration value\r\n\r\nFor further information have a look at:\r\nhttps://github.com/doctrine/DoctrineBundle/issues/673 at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\doctrine-bundle\\ConnectionFactory.php:79, Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:93, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:31, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:27)"} []
[2024-03-22 14:49:59] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\DBALException: "An exception occured while establishing a connection to figure out your platform version. You can circumvent this by setting a 'server_version' configuration value  For further information have a look at: https://github.com/doctrine/DoctrineBundle/issues/673" at C:\xampp\htdocs\svecommerce\vendor\doctrine\doctrine-bundle\ConnectionFactory.php line 79 {"exception":"[object] (Doctrine\\DBAL\\DBALException(code: 0): An exception occured while establishing a connection to figure out your platform version.\r\nYou can circumvent this by setting a 'server_version' configuration value\r\n\r\nFor further information have a look at:\r\nhttps://github.com/doctrine/DoctrineBundle/issues/673 at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\doctrine-bundle\\ConnectionFactory.php:79, Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:93, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:31, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:27)"} []
[2024-03-22 14:49:59] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\DBALException: An exception occured while establishing a connection to figure out your platform version. You can circumvent this by setting a 'server_version' configuration value  For further information have a look at: https://github.com/doctrine/DoctrineBundle/issues/673 at C:\xampp\htdocs\svecommerce\vendor\doctrine\doctrine-bundle\ConnectionFactory.php line 79) {"exception":"[object] (Doctrine\\DBAL\\DBALException(code: 0): An exception occured while establishing a connection to figure out your platform version.\r\nYou can circumvent this by setting a 'server_version' configuration value\r\n\r\nFor further information have a look at:\r\nhttps://github.com/doctrine/DoctrineBundle/issues/673 at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\doctrine-bundle\\ConnectionFactory.php:79, Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:93, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:31, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:27)"} []
[2024-03-22 15:16:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\DBALException: "An exception occured while establishing a connection to figure out your platform version. You can circumvent this by setting a 'server_version' configuration value  For further information have a look at: https://github.com/doctrine/DoctrineBundle/issues/673" at C:\xampp\htdocs\svecommerce\vendor\doctrine\doctrine-bundle\ConnectionFactory.php line 79 {"exception":"[object] (Doctrine\\DBAL\\DBALException(code: 0): An exception occured while establishing a connection to figure out your platform version.\r\nYou can circumvent this by setting a 'server_version' configuration value\r\n\r\nFor further information have a look at:\r\nhttps://github.com/doctrine/DoctrineBundle/issues/673 at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\doctrine-bundle\\ConnectionFactory.php:79, Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:93, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:31, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:27)"} []
[2024-03-22 15:16:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\DBALException: An exception occured while establishing a connection to figure out your platform version. You can circumvent this by setting a 'server_version' configuration value  For further information have a look at: https://github.com/doctrine/DoctrineBundle/issues/673 at C:\xampp\htdocs\svecommerce\vendor\doctrine\doctrine-bundle\ConnectionFactory.php line 79) {"exception":"[object] (Doctrine\\DBAL\\DBALException(code: 0): An exception occured while establishing a connection to figure out your platform version.\r\nYou can circumvent this by setting a 'server_version' configuration value\r\n\r\nFor further information have a look at:\r\nhttps://github.com/doctrine/DoctrineBundle/issues/673 at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\doctrine-bundle\\ConnectionFactory.php:79, Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:93, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:31, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:27)"} []
[2024-03-22 15:20:17] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\DBALException: "An exception occured while establishing a connection to figure out your platform version. You can circumvent this by setting a 'server_version' configuration value  For further information have a look at: https://github.com/doctrine/DoctrineBundle/issues/673" at C:\xampp\htdocs\svecommerce\vendor\doctrine\doctrine-bundle\ConnectionFactory.php line 79 {"exception":"[object] (Doctrine\\DBAL\\DBALException(code: 0): An exception occured while establishing a connection to figure out your platform version.\r\nYou can circumvent this by setting a 'server_version' configuration value\r\n\r\nFor further information have a look at:\r\nhttps://github.com/doctrine/DoctrineBundle/issues/673 at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\doctrine-bundle\\ConnectionFactory.php:79, Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:93, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:31, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:27)"} []
[2024-03-22 15:20:17] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\DBALException: An exception occured while establishing a connection to figure out your platform version. You can circumvent this by setting a 'server_version' configuration value  For further information have a look at: https://github.com/doctrine/DoctrineBundle/issues/673 at C:\xampp\htdocs\svecommerce\vendor\doctrine\doctrine-bundle\ConnectionFactory.php line 79) {"exception":"[object] (Doctrine\\DBAL\\DBALException(code: 0): An exception occured while establishing a connection to figure out your platform version.\r\nYou can circumvent this by setting a 'server_version' configuration value\r\n\r\nFor further information have a look at:\r\nhttps://github.com/doctrine/DoctrineBundle/issues/673 at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\doctrine-bundle\\ConnectionFactory.php:79, Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:93, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:31, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'u672279739_salvacero'@'localhost' (using password: YES) at C:\\xampp\\htdocs\\svecommerce\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:27)"} []
[2024-03-22 15:31:33] request.CRITICAL: Uncaught PHP Exception SmartyException: "unable to write file C:\xampp\htdocs\svecommerce\var\cache\prod\smarty\compile\75\97\2f\75972fac8c40051b7d0edf4a84a36529bd8c0cb0_0.file.login_addons.tpl.php" at C:\xampp\htdocs\svecommerce\vendor\smarty\smarty\libs\sysplugins\smarty_internal_runtime_writefile.php line 90 {"exception":"[object] (SmartyException(code: 0): unable to write file C:\\xampp\\htdocs\\svecommerce\\var\\cache\\prod\\smarty\\compile\\75\\97\\2f\\75972fac8c40051b7d0edf4a84a36529bd8c0cb0_0.file.login_addons.tpl.php at C:\\xampp\\htdocs\\svecommerce\\vendor\\smarty\\smarty\\libs\\sysplugins\\smarty_internal_runtime_writefile.php:90)"} []
[2024-03-22 15:31:33] request.CRITICAL: Uncaught PHP Exception SmartyException: "unable to write file C:\xampp\htdocs\svecommerce\var\cache\prod\smarty\compile\75\97\2f\75972fac8c40051b7d0edf4a84a36529bd8c0cb0_0.file.login_addons.tpl.php" at C:\xampp\htdocs\svecommerce\vendor\smarty\smarty\libs\sysplugins\smarty_internal_runtime_writefile.php line 90 {"exception":"[object] (SmartyException(code: 0): unable to write file C:\\xampp\\htdocs\\svecommerce\\var\\cache\\prod\\smarty\\compile\\75\\97\\2f\\75972fac8c40051b7d0edf4a84a36529bd8c0cb0_0.file.login_addons.tpl.php at C:\\xampp\\htdocs\\svecommerce\\vendor\\smarty\\smarty\\libs\\sysplugins\\smarty_internal_runtime_writefile.php:90)"} []
[2024-03-22 15:31:34] cache.WARNING: Failed to save key "%5BPrestaShopBundle%5CEntity%5CAdminFilter%5D%5B1%5D" (array) {"key":"%5BPrestaShopBundle%5CEntity%5CAdminFilter%5D%5B1%5D","type":"array","exception":"[object] (ErrorException(code: 0): rename(C:\\xampp\\htdocs\\svecommerce\\var\\cache\\prod\\pools\\Bxo9eUMTV3\\65fdeaa60b0eb4.83397115,C:\\xampp\\htdocs\\svecommerce\\var\\cache\\prod\\pools\\Bxo9eUMTV3\\T\\L\\1nJaYXc-RB+Wx08hHot-): Acceso denegado. (code: 5) at C:\\xampp\\htdocs\\svecommerce\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Cache\\Traits\\FilesystemCommonTrait.php:93)"} []
[2024-03-22 15:31:34] request.CRITICAL: Exception thrown when handling an exception (RuntimeException: Failed to write cache file "C:\xampp\htdocs\svecommerce\var\cache\prod/twig/50/502b4e3218eb3447b57a7825196369f040b78e6781556dc3516e2802911692b7.php". at C:\xampp\htdocs\svecommerce\vendor\twig\twig\src\Cache\FilesystemCache.php line 80) {"exception":"[object] (RuntimeException(code: 0): Failed to write cache file \"C:\\xampp\\htdocs\\svecommerce\\var\\cache\\prod/twig/50/502b4e3218eb3447b57a7825196369f040b78e6781556dc3516e2802911692b7.php\". at C:\\xampp\\htdocs\\svecommerce\\vendor\\twig\\twig\\src\\Cache\\FilesystemCache.php:80)"} []
[2024-03-22 17:03:50] request.CRITICAL: Uncaught PHP Exception SmartyException: "unable to write file C:\xampp\htdocs\svecommerce\var\cache\prod\smarty\compile\75\97\2f\75972fac8c40051b7d0edf4a84a36529bd8c0cb0_0.file.login_addons.tpl.php" at C:\xampp\htdocs\svecommerce\vendor\smarty\smarty\libs\sysplugins\smarty_internal_runtime_writefile.php line 90 {"exception":"[object] (SmartyException(code: 0): unable to write file C:\\xampp\\htdocs\\svecommerce\\var\\cache\\prod\\smarty\\compile\\75\\97\\2f\\75972fac8c40051b7d0edf4a84a36529bd8c0cb0_0.file.login_addons.tpl.php at C:\\xampp\\htdocs\\svecommerce\\vendor\\smarty\\smarty\\libs\\sysplugins\\smarty_internal_runtime_writefile.php:90)"} []
[2024-03-22 17:03:57] request.CRITICAL: Uncaught PHP Exception SmartyException: "unable to write file C:\xampp\htdocs\svecommerce\var\cache\prod\smarty\compile\75\97\2f\75972fac8c40051b7d0edf4a84a36529bd8c0cb0_0.file.login_addons.tpl.php" at C:\xampp\htdocs\svecommerce\vendor\smarty\smarty\libs\sysplugins\smarty_internal_runtime_writefile.php line 90 {"exception":"[object] (SmartyException(code: 0): unable to write file C:\\xampp\\htdocs\\svecommerce\\var\\cache\\prod\\smarty\\compile\\75\\97\\2f\\75972fac8c40051b7d0edf4a84a36529bd8c0cb0_0.file.login_addons.tpl.php at C:\\xampp\\htdocs\\svecommerce\\vendor\\smarty\\smarty\\libs\\sysplugins\\smarty_internal_runtime_writefile.php:90)"} []
[2024-03-22 17:03:57] request.CRITICAL: Uncaught PHP Exception SmartyException: "unable to write file C:\xampp\htdocs\svecommerce\var\cache\prod\smarty\compile\75\97\2f\75972fac8c40051b7d0edf4a84a36529bd8c0cb0_0.file.login_addons.tpl.php" at C:\xampp\htdocs\svecommerce\vendor\smarty\smarty\libs\sysplugins\smarty_internal_runtime_writefile.php line 90 {"exception":"[object] (SmartyException(code: 0): unable to write file C:\\xampp\\htdocs\\svecommerce\\var\\cache\\prod\\smarty\\compile\\75\\97\\2f\\75972fac8c40051b7d0edf4a84a36529bd8c0cb0_0.file.login_addons.tpl.php at C:\\xampp\\htdocs\\svecommerce\\vendor\\smarty\\smarty\\libs\\sysplugins\\smarty_internal_runtime_writefile.php:90)"} []
[2024-03-24 13:00:17] request.CRITICAL: Uncaught PHP Exception SmartyException: "Invalid compiled template for 'controllers/modules/login_addons.tpl'" at C:\xampp\htdocs\svecommerce\vendor\smarty\smarty\libs\sysplugins\smarty_template_resource_base.php line 116 {"exception":"[object] (SmartyException(code: 0): Invalid compiled template for 'controllers/modules/login_addons.tpl' at C:\\xampp\\htdocs\\svecommerce\\vendor\\smarty\\smarty\\libs\\sysplugins\\smarty_template_resource_base.php:116)"} []
[2024-03-24 15:17:04] request.CRITICAL: Uncaught PHP Exception SmartyException: "Invalid compiled template for 'controllers/modules/login_addons.tpl'" at C:\xampp\htdocs\svecommerce\vendor\smarty\smarty\libs\sysplugins\smarty_template_resource_base.php line 116 {"exception":"[object] (SmartyException(code: 0): Invalid compiled template for 'controllers/modules/login_addons.tpl' at C:\\xampp\\htdocs\\svecommerce\\vendor\\smarty\\smarty\\libs\\sysplugins\\smarty_template_resource_base.php:116)"} []
[2024-03-24 15:17:09] request.CRITICAL: Uncaught PHP Exception SmartyException: "unable to write file C:\xampp\htdocs\svecommerce\var\cache\prod\smarty\compile\75\97\2f\75972fac8c40051b7d0edf4a84a36529bd8c0cb0_0.file.login_addons.tpl.php" at C:\xampp\htdocs\svecommerce\vendor\smarty\smarty\libs\sysplugins\smarty_internal_runtime_writefile.php line 90 {"exception":"[object] (SmartyException(code: 0): unable to write file C:\\xampp\\htdocs\\svecommerce\\var\\cache\\prod\\smarty\\compile\\75\\97\\2f\\75972fac8c40051b7d0edf4a84a36529bd8c0cb0_0.file.login_addons.tpl.php at C:\\xampp\\htdocs\\svecommerce\\vendor\\smarty\\smarty\\libs\\sysplugins\\smarty_internal_runtime_writefile.php:90)"} []
[2024-03-24 15:17:09] request.CRITICAL: Uncaught PHP Exception SmartyException: "unable to write file C:\xampp\htdocs\svecommerce\var\cache\prod\smarty\compile\75\97\2f\75972fac8c40051b7d0edf4a84a36529bd8c0cb0_0.file.login_addons.tpl.php" at C:\xampp\htdocs\svecommerce\vendor\smarty\smarty\libs\sysplugins\smarty_internal_runtime_writefile.php line 90 {"exception":"[object] (SmartyException(code: 0): unable to write file C:\\xampp\\htdocs\\svecommerce\\var\\cache\\prod\\smarty\\compile\\75\\97\\2f\\75972fac8c40051b7d0edf4a84a36529bd8c0cb0_0.file.login_addons.tpl.php at C:\\xampp\\htdocs\\svecommerce\\vendor\\smarty\\smarty\\libs\\sysplugins\\smarty_internal_runtime_writefile.php:90)"} []
[2024-03-24 15:39:44] request.CRITICAL: Uncaught PHP Exception SmartyException: "Invalid compiled template for 'controllers/modules/login_addons.tpl'" at C:\xampp\htdocs\svecommerce\vendor\smarty\smarty\libs\sysplugins\smarty_template_resource_base.php line 116 {"exception":"[object] (SmartyException(code: 0): Invalid compiled template for 'controllers/modules/login_addons.tpl' at C:\\xampp\\htdocs\\svecommerce\\vendor\\smarty\\smarty\\libs\\sysplugins\\smarty_template_resource_base.php:116)"} []
[2024-03-24 16:12:02] request.CRITICAL: Uncaught PHP Exception SmartyException: "Invalid compiled template for 'controllers/modules/login_addons.tpl'" at C:\xampp\htdocs\svecommerce\vendor\smarty\smarty\libs\sysplugins\smarty_template_resource_base.php line 116 {"exception":"[object] (SmartyException(code: 0): Invalid compiled template for 'controllers/modules/login_addons.tpl' at C:\\xampp\\htdocs\\svecommerce\\vendor\\smarty\\smarty\\libs\\sysplugins\\smarty_template_resource_base.php:116)"} []
[2024-03-24 16:14:24] request.CRITICAL: Uncaught PHP Exception SmartyException: "Invalid compiled template for 'controllers/modules/login_addons.tpl'" at C:\xampp\htdocs\svecommerce\vendor\smarty\smarty\libs\sysplugins\smarty_template_resource_base.php line 116 {"exception":"[object] (SmartyException(code: 0): Invalid compiled template for 'controllers/modules/login_addons.tpl' at C:\\xampp\\htdocs\\svecommerce\\vendor\\smarty\\smarty\\libs\\sysplugins\\smarty_template_resource_base.php:116)"} []
[2024-03-24 16:14:29] request.CRITICAL: Uncaught PHP Exception SmartyException: "Invalid compiled template for 'controllers/modules/login_addons.tpl'" at C:\xampp\htdocs\svecommerce\vendor\smarty\smarty\libs\sysplugins\smarty_template_resource_base.php line 116 {"exception":"[object] (SmartyException(code: 0): Invalid compiled template for 'controllers/modules/login_addons.tpl' at C:\\xampp\\htdocs\\svecommerce\\vendor\\smarty\\smarty\\libs\\sysplugins\\smarty_template_resource_base.php:116)"} []
[2024-03-24 16:18:16] request.CRITICAL: Uncaught PHP Exception SmartyException: "Invalid compiled template for 'controllers/modules/login_addons.tpl'" at C:\xampp\htdocs\svecommerce\vendor\smarty\smarty\libs\sysplugins\smarty_template_resource_base.php line 116 {"exception":"[object] (SmartyException(code: 0): Invalid compiled template for 'controllers/modules/login_addons.tpl' at C:\\xampp\\htdocs\\svecommerce\\vendor\\smarty\\smarty\\libs\\sysplugins\\smarty_template_resource_base.php:116)"} []
[2024-03-24 16:18:16] request.CRITICAL: Uncaught PHP Exception SmartyException: "unable to write file C:\xampp\htdocs\svecommerce\var\cache\prod\smarty\compile\75\97\2f\75972fac8c40051b7d0edf4a84a36529bd8c0cb0_0.file.login_addons.tpl.php" at C:\xampp\htdocs\svecommerce\vendor\smarty\smarty\libs\sysplugins\smarty_internal_runtime_writefile.php line 90 {"exception":"[object] (SmartyException(code: 0): unable to write file C:\\xampp\\htdocs\\svecommerce\\var\\cache\\prod\\smarty\\compile\\75\\97\\2f\\75972fac8c40051b7d0edf4a84a36529bd8c0cb0_0.file.login_addons.tpl.php at C:\\xampp\\htdocs\\svecommerce\\vendor\\smarty\\smarty\\libs\\sysplugins\\smarty_internal_runtime_writefile.php:90)"} []
